README.md

```diff
@@ -18,44 +18,91 @@ npm start
 
-1. Choose a DataObject to store our Fluid state
-2. Create a container to hold an instance of that Fluid state
-3. Write a view to render and modify that Fluid state
+1. Choose a Fluid data structure to store our Fluid state.
+2. Create a container to hold an instance of that Fluid state.
+3. Write a view to render and modify that Fluid state.
 
-### Choosing a DataObject
+### Choosing a Data Structure
 
-A Fluid [DataObject](https://fluidframework.com/docs/glossary/#dataobject) is "designed to organize distributed data structures into semantically meaningful groupings for your scenario, as well as, providing an API surface to your data".
+Fluid's distributed data structures underpin all Fluid apps. While distributed data structures are low-level data
+structures, they provide familiar APIs and consistent merge behavior, and can serve a number of needs directly.
 
-In subsequent demos we will look at building custom DataObjects, but for many use cases (including this dice roller) the built in KeyValuePair DataObject will work perfectly.
+Fluid also provides DataObjects, which are designed to organize distributed data structures into semantically meaningful
+groupings as well as provide an API surface to your data.
 
-The KVPair DataObject provides the basic functionality to set and get data of any type on a given key, and inform your application when the data in that key changes.
+In subsequent demos we will look at building custom DataObjects, but for many use cases (including this dice roller) the
+built in distributed data structures will work perfectly.
+
+#### Using SharedMap Distributed Data Structure
+
+The SharedMap distributed data structure provides the basic functionality to set and get data of any JSON-serializable
+type on a given key, and inform your application when the data in that key changes.
 
-#### Using KeyValuePair DataObject
 
 ```ts
 // Set the string 'some data' to the key 'MyDataKey'
-kvpair.set('MyDataKey', 'some data');
+myMap.set('MyDataKey', 'some data');
 
 // Retrieve the string stored in 'MyDataKey'
-kvpair.get('MyDataKey'); // returns string 'some data'
+myMap.get('MyDataKey'); // returns string 'some data'
 
 // Set a 'changed' event that logs the event when any data changes
-kvpair.on('changed', (event) => console.log(event));
+myMap.on('changed', (event) => console.log(event));
 
 // Changed callback is called each time a key's value is changed
-kvpair.set('MyDataKey', 'some new data'); // logs { key: 'MyDataKey', path: '/', previousValue: 'some data' }
+myMap.set('MyDataKey', 'some new data'); // logs { key: 'MyDataKey', path: '/', previousValue: 'some data' }
 ```
 
 ### Creating a Fluid Container
 
-Fluid [Containers](https://fluidframework.com/docs/glossary/#container) are your applications entry point to the Fluid Framework.
+[Fluid containers](https://fluidframework.com/docs/glossary/#container) are your application's entry point to the Fluid
```

Comment:
nit: Would it be confusing for devs reading this to be pointed to documentation for the runtime-level `Container`, when TinylicousClient will give them back a `FluidContainer`? Terminology wise, they may be wondering why the two objects to match.